src/nvim/ex_cmds_defs.h

```diff
@@ -71,6 +71,14 @@
 #define WORD1   (EXTRA | NOSPC) /* one extra word allowed */
 #define FILE1   (FILES | NOSPC) /* 1 file allowed, defaults to current file */
 
+// values for cmd_addr_type 
+#define ADDR_LINES              0
+#define ADDR_WINDOWS            1
+#define ADDR_ARGUMENTS          2
+#define ADDR_LOADED_BUFFERS     3
```

Comment:
Not sure about this name change. I would prefer to make it sense as english instead.